Extension/src/LanguageServer/extension.ts

```diff
@@ -594,8 +600,55 @@ async function onRunCodeAnalysisOnAllFiles(): Promise<void> {
     getActiveClient().handleRunCodeAnalysisOnAllFiles();
 }
 
-async function onClearCodeAnalysisSquiggles(): Promise<void> {
-    getActiveClient().handleClearCodeAnalysisSquiggles();
+async function onRemoveAllCodeAnalysisProblems(): Promise<void> {
+    getActiveClient().handleRemoveAllCodeAnalysisProblems();
+}
+
+async function onRemoveCodeAnalysisProblems(refreshSquigglesOnSave: boolean, identifiersAndUris: CodeAnalysisDiagnosticIdentifiersAndUri[]): Promise<void> {
+    getActiveClient().handleRemoveCodeAnalysisProblems(refreshSquigglesOnSave, identifiersAndUris);
+}
+
+// Needed due to https://github.com/microsoft/vscode/issues/148723 .
+const codeActionAbortedString: string = localize('code.action.aborted', "The code action fix was skipped because it was generated from an older document version.");
```

Comment:
Should this refer to a "code analysis fix" or a "quick fix"?  It's unclear to me what the ideal terminology would be.  I see both "code action" and "quick fix" referred to in the VS Code UI, but here the context seems to be specific code analysis.

Reply:
I'm not sure. I think "fix" is understandable, and I'm not sure how "quick fix" would be more understandable. I know VS Code has a "quick fix" type that the code action has. "Code action quick fix" is another possibility. I could just use "quick fix" if we wanted the shorted amount of text.

Reply:
How about `code analysis fix`?  Also, the user might not be aware of the process of 'generating' code actions or of document versions.  How about "The fix could not be applied because the document has changed" ?

Reply:
"The code analysis fix could not be applied because the document has changed" is simple and clear. It would be helpful to add "code analysis" to let the user know what type of fix failed to apply.

Reply:
Mentioning "code analysis" seems redundant/obvious, because the user would have had to have just clicked the "Fix this <clang-tidy type> message". However, I'm not opposed to being more explicit in the message.

Reply:
I vote for explicit message just in case.

Reply:
Sure